pandas/io/pytables.py

```diff
@@ -1133,6 +1133,10 @@ def _create_storer(self, group, format=None, value=None, append=False,
                        **kwargs):
         """ return a suitable class to operate """
 
+        if isinstance(value, Series) and any(np.iscomplex(value)) and format=='table':
```

Comment:
you can if needed but not why it matters if it's s Series or not - complex either works for s block or it doesn't (and it should)

Series are stored as Dataframes actually